package.json

```diff
@@ -42,24 +42,32 @@
   "devDependencies": {
     "@types/chai": "^4.2.11",
     "@types/mocha": "^7.0.2",
-    "@types/node": "^13.13.2",
+    "@types/node": "^13.13.4",
+    "@types/node-fetch": "^2.5.7",
     "browserify": "^16.5.1",
+    "browserify-shim": "^3.8.14",
     "chai": "^4.2.0",
+    "cross-fetch": "^3.0.4",
     "envify": "^4.1.0",
     "karma": "^5.0.2",
     "karma-chai": "^0.1.0",
     "karma-chrome-launcher": "^3.1.0",
     "karma-mocha": "^2.0.0",
     "karma-mocha-reporter": "^2.2.5",
-    "mocha": "^7.1.1",
+    "mocha": "^7.1.2",
     "mocha-typescript": "^1.1.17",
     "tsify": "^4.0.1",
     "typescript": "^3.8.3",
     "uglify-js": "^3.9.1",
     "uglifyify": "^5.0.2"
   },
+  "browserify-shim": {
+    "node-fetch": "cross-fetch",
+    "form-data": "global:FormData"
+  },
```

Comment:
This is some magic handwaving but basically when we compile the browser version of this project we replace any imports of node-fetch with cross-fetch (so that we can use the default import and name it fetch). We also import form-data for nodejs's benefit and replace it at browser time with the `FormData` class in the browser.